README.md

```diff
@@ -10,15 +10,15 @@ This web app will serve as a self-serve customer portal for an existing app call
 - Tyler Mair: tylermair@cmail.carleton.ca
 
 ## Current State of the Project
```

Comment:
Please include that we ONLY support CHROME

Reply:
Added